style.css

```diff
@@ -0,0 +1,180 @@
+
+* {
+    box-sizing: border-box;
+    font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif; 
+    margin: 0;
+    padding: 0;
+}
+/* pictures as a background-image inside the grid: */
+.all-pictures {
+    display: grid;
+    grid-column-gap: 6px;
+    grid-row-gap: 5px;
+    grid-template-columns: repeat(4, 1fr);
+}
+.bigger-fig {
+    background-image: url("images/fig-one.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover; 
+    grid-column-start: 1;
+    grid-column-end: 5;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    height:690px;
+}
+.medium-fig-2 {
+    background-image: url("images/fig-two.jpg");
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+.medium-fig-3 {
+    background-image: url("images/fig-three.jpg");
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+.medium-fig-4 {
+    background-image: url("images/fig-four.jpg");
+    grid-column-start: 3;
+    grid-column-end: 4;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+.medium-fig-5 {
+    background-image: url("images/fig-five.jpg");
+    grid-column-start: 4;
+    grid-column-end: 5;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+/*nav bar: */
+.nav-bar-apple {    
+    left: 10%;  
+    margin-top: 5%;
+    position: absolute;
+    width: 80%;
+}
+.nav-bar-apple li {
+    
+    border-right: 1px solid grey;
+    display: table-cell;
+    height: 36px;
+    overflow: hidden;      
+    text-align: center;
+    vertical-align: middle;
+    width: 7%;    
+}
+.nav-bar-apple ul {
+    list-style-type: none;
+    background-image: linear-gradient(to bottom, #606060 50%, #707070 50%);
+    opacity: 1;
```

Comment:
`SUGGESTION`

Consider adding border-radius `border-radius: 4px;` to make it look more like the original page navbar.

Reply:
Border-radius added :) Thanks for the feedback!!!